### PR TITLE
Added a README.md file

### PR DESCRIPTION
**Pull request by Abhinav-ca**
adding a README file that was missing for the project

### CODE REVIEW

**Review by Abhinav-fake — CHANGES_REQUESTED** [review]
this is too short. it would be nice to see a example

**Review by Abhinav-fake — COMMENTED** [review]
make it in one commit

**Review by Abhinav-fake — CHANGES_REQUESTED** [review]
ok

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Rearrange
+
+This module is used for rearranging names
+turns LastName, FirstName into FirstName LastName
+
```
Comment (Abhinav-fake) [inline README.md:5]: this line is missing period

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Rearrange
```
Comment (Abhinav-fake) [inline README.md:1]: it coll but need more examples

File: README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Rearrange
+
+This module is used for rearranging names
+turns LastName, FirstName into FirstName LastName
+
+# Example
+calling `rearrangename("Turing, Alan")` as a parameter, will return `Alan Turing`
```
Comment (Abhinav-fake) [inline README.md]: it cool but need more examples

File: README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Rearrange
+
+This module is used for rearranging names
+turns LastName, FirstName into FirstName LastName
+
+# Example
```
Comment (Abhinav-fake) [inline README.md]: write ## to make it smaller

**Comment by Abhinav-ca** [comment]
resolved all coment. And please take another look

**Pull request closed without merging**